foundry.toml

```diff
@@ -8,22 +8,26 @@ fs_permissions = [
     { access = "read", path = "./out"},
     { access = "read-write", path = "./deployments"},
     { access = "read", path = "./actual-deployments"},
+    { access = "read-write", path = "./cache" }
 ]
 gas_reports = ["DcntEth", "DecentEthRouter"]
+build_info = true
+extra_output = ['storageLayout']
+always_use_create_2_factory = true
 
 [rpc_endpoints]
 ethereum = "${ETHEREUM_RPC}"
 arbitrum = "${ARBITRUM_RPC}"
 optimism = "${OPTIMISM_RPC}"
 polygon = "${POLYGON_RPC}"
 avalanche = "${AVALANCHE_RPC}"
-ftm-testnet = "${FTM_TESTNET_RPC}"
-fuji = "${FUJI_RPC}"
+fantom_testnet = "${FTM_TESTNET_RPC}"
+avalanche_fuji = "${FUJI_RPC}"
```

Comment:
Changed these RPC endpoint names because Sphinx expects that they have specific names